.env

```diff
@@ -0,0 +1,101 @@
+### Documentation available at https://wodby.com/docs/stacks/wordpress/local
+### Changelog can be found at https://github.com/wodby/docker4wordpress/releases
+### Images tags format explained at https://github.com/wodby/docker4wordpress#images-tags
+
+### PROJECT SETTINGS
+
+DOCROOT=/var/www/html
+BASE_DIR=/opt/projects/mollie/WooCommerce
+PROJECT_NAME=my_wordpress_project
+PROJECT_BASE_URL=wp.docker.localhost
+PROJECT_MOUNT_PATH=/var/www/html/wp-content/plugins/WooCommerce
+
+DB_NAME=wordpress
+DB_USER=wordpress
+DB_PASSWORD=wordpress
+DB_ROOT_PASSWORD=password
+DB_HOST=mariadb
+
+### --- MARIADB ----
+
+MARIADB_TAG=10.3-3.4.11
+#MARIADB_TAG=10.2-3.4.11
+#MARIADB_TAG=10.1-3.4.11
+
+### --- VANILLA WORDPRESS ----
+
+WORDPRESS_TAG=5-4.16.1
+
+### --- PHP ----
+
+# Linux (uid 1000 gid 1000)
+
+PHP_TAG=7.3-dev-4.12.12
```

Comment:
Would not better to enable php 5.6 since the plugin must be compatible with that version?

Reply:
This is counter-intuitive, but if you look in the override file, you will see that it uses the WordPress image, not the PHP one. This is currently the last configuration that worked, and with it you cannot switch the PHP version; only WP version. However, the dependencies are installed with PHP 5.6.40.